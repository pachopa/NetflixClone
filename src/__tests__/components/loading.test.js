import React from "react";
import { getByTestId, render } from "@testing-library/react";
import { Loading } from "../../components";

describe("<Loading />", () => {
  it("renders the <Loading /> component", () => {
    const { container, getByTestId } = render(
      <Loading src="/images/chris.png" data-testid="loading" />
    );

    expect(getByTestId("loading")).toBeTruthy();
    expect(getByTestId("loading-picture")).toBeTruthy();
    expect(container.firstChild).toMatchSnapshot();
  });

  it("redners the <Loading.ReleaseBody />", () => {
    const { container, getByTestId } = render(<Loading.ReleaseBody />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
