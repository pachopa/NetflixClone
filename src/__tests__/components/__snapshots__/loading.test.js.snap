// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> redners the <Loading.ReleaseBody /> 1`] = `null`;

exports[`<Loading /> renders the <Loading /> component 1`] = `
<div
  class="sc-bdnylx ghrSFu"
  data-testid="loading"
>
  <img
    class="sc-gtssRu dbobBa"
    data-testid="loading-picture"
    src="/images/users//images/chris.png.png"
  />
</div>
`;
